src/hearings/containers/request-hearing/request-hearing.component.html

```diff
@@ -7,7 +7,7 @@
       <button *ngIf="isChildPage" class="govuk-button" data-module="govuk-button" (click)="onContinue()">{{ 'Continue' | rpxTranslate }}</button>
       <button *ngIf="isCreateEditSummary" class="govuk-button" data-module="govuk-button" (click)="submitNewRequest()">{{ 'Submit request' | rpxTranslate }}</button>
       <button *ngIf="isViewEditSummary || isEditSummary" class="govuk-button" data-module="govuk-button" (click)="submitUpdatedRequest()">{{ 'Submit updated request' | rpxTranslate }}</button>
-      <button *ngIf="isViewEditReason" class="govuk-button" data-module="govuk-button" (click)="submitChangeRequest()">{{ 'Submit change request' | rpxTranslate }}</button>
+      <button *ngIf="isViewEditReason" [disabled]="hasSubmitted" class="govuk-button" data-module="govuk-button" (click)="submitChangeRequest()">{{ 'Submit change request' | rpxTranslate }}</button>
```

Comment:
If it starts disabled, where is the code to enable it?

Reply:
The button starts enabled and is then disabled (within the component.ts) once submit is attempted (the button has been clicked)